src/lib.rs

```diff
@@ -38,7 +38,12 @@ use mechanics::{AuthorMode, PagesChapterMode};
 
 use paste::paste;
 
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
 /// A fully parsed bibliography.
+/// 
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[derive(Debug, Clone)]
```

Comment:
Switch the derives as above. Is there a reason for the additional blank doc-comment line?